mathesar_ui/src/systems/users-and-permissions/UserPermissionForm.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+  import { Help } from '@mathesar-component-library';
+  import WarningBox from '@mathesar/components/message-boxes/WarningBox.svelte';
+  import type { UserModel } from '@mathesar/stores/users';
+  import { getSchemasStoreForDB } from '@mathesar/stores/schemas';
+  import { getUsersStoreFromContext } from '@mathesar/stores/users'; 
+  import type { Database, SchemaEntry } from '@mathesar/AppTypes';
+
+  export let database : Database;
+  export let schema : SchemaEntry;
+  export let userId: UserModel['id']

```

Comment:
This component accepts database and schema but the values are not passed when calling the component instance in EditUserPage. So these values will be undefined.

Please check out [Svelte docs](https://svelte.dev/docs#template-syntax-attributes-and-props) on how to work with components and props.